docs/examples/merging/updates/ghana_tender1.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": "1.1",
+  "publishedDate": "2020-10-21T13:02:26Z",
+  "publisher": {
+    "name": "Public Procurement Authority",
+    "scheme": "GH-RGD",
+    "uid": "1234 ",
+    "uri": "https://egovonline.gegov.gov.gh/RGDPortalWeb/portal/RGDHome/eghana.portal"
+  },
+  "uri": "https://www.ghaneps.gov.gh/ocds/services/releasepackage/getfullreleasepackageforocid/ocds-uhveoc-144560",
+  "releases": [
+    {
+      "ocid": "ocds-uhveoc-144560",
+      "id": "ocds-uhveoc-144560-1601380494627-tender",
+      "date": "2020-10-21T13:02:26Z",
+      "tag": [
+        "tender"
+      ],
+      "initiationType": "tender",
+      "parties": [
+        {
+          "name": "ECONOMIC AND ORGANISED CRIME OFFICE",
+          "id": "14957",
+          "roles": [
+            "buyer",
+            "procuringEntity"
+          ]
+        }
+      ],
+      "tender": {
+        "id": "144560",
+        "title": "Toners",
+        "description": "The E.O.C.O is requesting for quotations from your outfit for the supply of ten (10) pieces each of 26A and 29A toners.\r\nYou are requested to submit your quotation via GHANEPS portal.\r\nDeadline for submission is 30th October, 2020. at 16:00 HRS GMT.",
+        "status": "active",
+        "value": {
+          "amount": 13000,
+          "currency": "GHS"
+        },
+        "mainProcurementCategory": "goods",
+        "numberOfTenderers": 0,
```

Comment:
but this is one of the fields that changes, in 2nd and 3rd release this number goes up to 2

Reply:
Let's choose a different field (one that is expected to be populated in a tender release) to demonstrate how a field's value is updated, e.g. `tender.value`.

Reply:
I do see the point of this but if we make this change then this becomes a completely fictional example (it's already a bit fictional due to the removal of 1 of the tenderers). Which is fine if the agreement is that we don't actually need "real" examples just ones that are coherent but if that's the case I think I should change the ocid as well so that it's not using a real ocid with different data from the example.

Reply:
It's okay to be fictional as long as it's potentially real (i.e. the change in tender value can actually occur).

Reply:
I've removed this field, changed the `tender.value` in the second release (and third and versioned) and also updated the ocid and `tender.id` so that they're now fictional

Reply:
Resolving as the commented file no longer appears in this PR.